src/graph/Ingredient/IngredientMutations.js

```diff
@@ -1,8 +1,7 @@
-const graphql = require('graphql');
-const { GraphQLObjectType, GraphQLString, GraphQLNonNull } = graphql;
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull } from 'graphql';
 
-const IngredientType = require('./IngredientType');
-const { editIngredient, addNewIngredient } = require('./IngredientService');
+import IngredientType from './IngredientType';
+import { editIngredient, addNewIngredient } from './IngredientService';
 
 const IngredientMutations = {
```

Comment:
You can export default right here or pre-define a const IngredientMutations. Your choice.